src/RecyclableMemoryStream.cs

```diff
@@ -1538,7 +1538,10 @@ private void EnsureCapacity(long newCapacity)
             if (newCapacity > this.memoryManager.MaximumStreamCapacity && this.memoryManager.MaximumStreamCapacity > 0)
             {
                 this.memoryManager.ReportStreamOverCapacity(this.id, this.tag, newCapacity, this.AllocationStack);
-                throw new OutOfMemoryException($"Requested capacity is too large: {newCapacity}. Limit is {this.memoryManager.MaximumStreamCapacity}.");
+
+                var requestedCapacity = newCapacity.ToString(CultureInfo.InvariantCulture);
+                var limit = this.memoryManager.MaximumStreamCapacity.ToString(CultureInfo.InvariantCulture);
+                throw new OutOfMemoryException($"Requested capacity is too large: {requestedCapacity}. Limit is {limit}.");
```

Comment:
There's no need to have variables for that.

What's wrong with this?

```csharp
throw new OutOfMemoryException($"Requested capacity is too large: {newCapacity.ToString(CultureInfo.InvariantCulture)}. Limit is {this.memoryManager.MaximumStreamCapacity.ToString(CultureInfo.InvariantCulture)}.");

Other than readability, that is. 😄 
```

Reply:
That's it, it's too long :)
I can inline them if you wish.

Reply:
@benmwatson, @grbell-ms, what do you think?

Reply:
I don't think we care if we do culture specific formatting or boxing in this rare error case. So let's keep it simple:

```cs
throw new OutOfMemoryException($"Requested capacity is too large: {newCapacity}. Limit is {this.memoryManager.MaximumStreamCapacity}.");
```

Reply:
I'm generally not in favor of introducing temp variables if I don't need them. I don't care about line length in this case. I'm open to culture formatting or not.

Reply:
> So let's keep it simple

Agreed! Done.